roles/wordpress-setup/templates/wordpress-site.conf.j2

```diff
@@ -38,7 +38,11 @@ server {
   include ssl.conf;
   include ssl-stapling.conf;
 
-  add_header Strict-Transport-Security "max-age=31536000; includeSubdomains; preload";
+  {% set hsts_max_age = item.value.ssl.hsts_max_age | default(nginx_hsts_max_age) %}
+  {% set hsts_include_subdomains = item.value.ssl.hsts_include_subdomains | default(nginx_hsts_include_subdomains) %}
+  {% set hsts_preload = item.value.ssl.hsts_preload | default(nginx_hsts_preload) %}
+
+  add_header Strict-Transport-Security "max-age={{ hsts_max_age }}{{ hsts_include_subdomains | ternary('; includeSubdomains', '') }}{{ hsts_preload | ternary('; preload', '') }}";
```

Comment:
The reason why I did the manual "; " prefix is because I was trying to follow the format:

![image](https://cloud.githubusercontent.com/assets/2192970/10594784/4dd88b28-7696-11e5-85c0-f0a545674125.png)

https://developer.mozilla.org/en-US/docs/Web/Security/HTTP_strict_transport_security#Enabling_Strict_Transport_Security

Other formatting examples

https://tools.ietf.org/html/draft-hodges-strict-transport-sec-02#page-17

https://www.owasp.org/index.php/HTTP_Strict_Transport_Security